pkg/secrets-store/nodeserver.go

```diff
@@ -240,9 +241,13 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 
 	notMnt, err = mount.New("").IsLikelyNotMountPoint(targetPath)
 	if err != nil {
-		log.Errorf("Error checking IsLikelyNotMountPoint: %v", err)
+		log.Errorf("Error checking targetPath %s IsLikelyNotMountPoint: %v", targetPath, err)
 		return nil, err
 	}
+	// this means the mount didn't succeed
+	if notMnt {
+		return nil, fmt.Errorf("after MountSecretsStoreObjectContent, notMnt: %v", notMnt)
+	}
 	log.Debugf("after MountSecretsStoreObjectContent, notMnt: %v", notMnt)
```

Comment:
since we are returning the error if not mounted, we can remove this log

Reply:
Done!